src/content/guidelines/grid.mdx

```diff
@@ -7,7 +7,7 @@ import GridPage from '../../components/guidelines/GridPage'
 
 # Grid
 
-The Zeppelin grid system provides a mobile-first and flexible layout system. The content is divided into an 12-column grid with defined breakpoints ensure optimized responsiveness and usability on a range of different devices. This overview helps designers and developers deliver clear, functional layouts.
+The Zeppelin grid system provides a mobile-first and flexible layout system. The content is divided into an 12-column grid with defined breakpoint ensure optimized responsiveness and usability on a range of different devices. This overview helps designers and developers deliver clear, functional layouts.
```

Comment:
This is not true :D - please remove the whole Grid section until we have a usable flex-box or css-grid Grid system!